genesyscloud/supported_content/resource_genesyscloud_supported_content_utils.go

```diff
@@ -0,0 +1,115 @@
+package supported_content
+
+import (
+	"terraform-provider-genesyscloud/genesyscloud/util/resourcedata"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v133/platformclientv2"
+)
+
+/*
+The resource_genesyscloud_supported_content_utils.go file contains various helper methods to marshal
+and unmarshal data into formats consumable by Terraform and/or Genesys Cloud.
+*/
+
+// getSupportedContentFromResourceData maps data from schema ResourceData object to a platformclientv2.Supportedcontent
+func getSupportedContentFromResourceData(d *schema.ResourceData) platformclientv2.Supportedcontent {
+	return platformclientv2.Supportedcontent{
+		Name:       platformclientv2.String(d.Get("name").(string)),
+		MediaTypes: buildMediaTypes(d.Get("media_types").([]interface{})),
+	}
+}
+
+// buildMediaTypes maps an []interface{} into a Genesys Cloud *[]platformclientv2.Mediatype
+func buildMediaTypes(mediaTypes []interface{}) *platformclientv2.Mediatypes {
+	var sdkMediaType platformclientv2.Mediatypes
+
+	for _, mediaType := range mediaTypes {
+		mediaTypesMap, ok := mediaType.(map[string]interface{})
+		if !ok {
+			continue
+		}
+
+		resourcedata.BuildSDKInterfaceArrayValueIfNotNil(&sdkMediaType.Allow, mediaTypesMap, "allow", buildMediaTypeAccesss)
+	}
+	return &sdkMediaType
+}
+
+// buildMediaTypeAccesss maps an []interface{} into a Genesys Cloud *[]platformclientv2.Mediatypeaccess
+func buildMediaTypeAccesss(mediaTypeAccesss []interface{}) *platformclientv2.Mediatypeaccess {
```

Comment:
`buildMediaTypeAccesss` -> `buildMediaTypeAccess`

Reply:
Also mediaTypeAccesss -> mediaTypeAccess

There are a few other instances below where there is an extra `s`